packages/config/src/options.ts

```diff
@@ -98,6 +98,11 @@ const isValidConfig = (key: string, config: any) => {
   return true
 }
 
+export const defaultSpecPattern = {
```

Comment:
Can we update the e2e & component configuration values to return these?

Reply:
It needs to be exported separately because it replaces [this](https://github.com/cypress-io/cypress/pull/20677/files#diff-635c05fe0a204b142186818a9bc2c31711cacf2059208cb8421b4eb264872c29) file that was removed - also we can't get the values from the resolved config bc there's no gonna be nested properties based on the testing type - just the top-level ones, so we do not now the default value if it's set on the config file and also we may not be able to know the default value of the testing type that is not being selected